pom.xml

```diff
@@ -184,9 +184,9 @@
             <scope>test</scope>
         </dependency>
         <dependency>
-            <groupId>net.rakugakibox.spring.boot</groupId>
+            <groupId>dev.akkinoc.spring.boot</groupId>
```

Comment:
Ser jo riktig ut dette.

Moved Maven group ID from "net.rakugakibox.spring.boot" to "dev.akkinoc.spring.boot". (https://github.com/akkinoc/logback-access-spring-boot-starter/pull/62)